core/src/main/java/com/datastrato/gravitino/catalog/CatalogManager.java

```diff
@@ -541,23 +558,27 @@ private CatalogWrapper loadCatalogInternal(NameIdentifier ident) throws NoSuchCa
     }
   }
 
-  private CatalogWrapper createCatalogWrapper(CatalogEntity entity) {
+  private CatalogWrapper createCatalogWrapper(CatalogEntity entity, CatalogWrapper oldWrapper) {
```

Comment:
It's weird to create a catalog wrapper with old catalog wrapper, can we define a new method?